brave/brave-ios
# Fix #8072: Portfolio v2

## Summary of Changes
- Merge NFT tab into Portfolio tab, and remove existing NFT tab from top page control
- Update Portfolio to use new v2 designs from Figma.
- Separated most views / content views outside of `PortfolioView`, header, segmented control, assets list, etc.

This pull request fixes #8072

## Submitter Checklist:


## Test Plan:
1. Open Portfolio, verify Buy / Send / Swap buttons are all opening their expected views
2. Tap the graph timeframe selector, verify graph updates when changed.
    - Price indicator should be green for balance increases, red for balance decreases (from start of graph timeframe)
3. Tap on the Assets / NFTs selector, verify displayed assets update to tapped on page as expected
4. Tap on individual assets & NFTs, verify their detail views open as expected
5. Switch to Assets section and open filters. Change filters to group by network or account & verify groups are outlined with a line to distinguish the groups when expanded and collapsed.


## Screenshots:
![Portfolio v2 - light](https://github.com/brave/brave-ios/assets/5314553/eaab85c0-df2c-4ce1-8932-3efba9cec7ee) | ![Portfolio assets v2 - light](https://github.com/brave/brave-ios/assets/5314553/dab37183-5f01-4587-a93e-d862d612ab7d) | ![Portfolio nfts v2 - light](https://github.com/brave/brave-ios/assets/5314553/5e627c1e-555f-4798-94be-5be6fe3389a1)
--|--|--
![Portfolio v2 - dark](https://github.com/brave/brave-ios/assets/5314553/cbe7ebc6-5e85-42be-9574-a7fe908a6f14) | ![Portfolio assets v2 - dark](https://github.com/brave/brave-ios/assets/5314553/d70ef67e-b3fd-45d1-b242-8f919034043b) | ![Portfolio nfts v2 - dark](https://github.com/brave/brave-ios/assets/5314553/91c7e599-fc71-4639-96bc-7b37e7fd341c)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
